### ISSUE TITLE
Why is motionEye loading just "jpeg images" as the feed? Why not using mpeg stream?

### ISSUE THREAD

**Issue by developerfromjokela** [ccrisan/motioneye#1915]
Hello!
I'm just thinking, motionEye currently loads just constantly a snapshot of the camera image. 
Why wouldn't we use mpeg streams? Why the extra code to constantly fetch images?

**Comment by developerfromjokela** [comment]
Hmm what is this? Ignoring user feedback?

**Comment by starbasessd** [comment]
No, trying to clean up OLD issues. Was going to go back and re-open labeled issues like bugs, questions, enhancement a feature requests. There hadn't been any feedback or comments since I labeled it a question back in November. Trying to make it easier for the team to find active issues, and stuff that hadn't been resolved by newer updates.

**Comment by developerfromjokela** [comment]
Ok
